src/app/services/enclosure/enclosure.octoprint.service.ts

```diff
@@ -307,6 +310,28 @@ export class EnclosureOctoprintService implements EnclosureService {
       .subscribe();
   }
 
+  private setPSUStateTuya(state: PSUState) {
+    const tuyaPayload: TuyaCommand = {
+      command: state === PSUState.ON ? 'turnOn' : 'turnOff',
+      label: this.configService.getTuyaLabel(),
+    };
+
+    this.http
+      .post(this.configService.getApiURL('plugin/tuyasmartplug'), tuyaPayload, this.configService.getHTTPHeaders())
+      .pipe(
+        catchError(error => {
+          this.notificationService.setNotification({
+            heading: $localize`:@@error-send-psu-gcode:Can't send GCode!`,
```

Comment:
```suggestion
            heading: $localize`:@@error-send-psu-command:Can't send plug command!`,
```

Reply:
Oops, this got updated in my fork but forgot to cherry-pick that fix, thanks!